Fetch Individual Student Result From VTU Website

Colin-Christon opened the pull request:
This PR introduces a new functionality for fetching individual student results from the VTU website using their USN. The changes include the addition of a new action, fetch_one, in scrapper.py, the implementation of the corresponding method.

Colin-Christon committed:
Add fetch_one command to scrapper.py
Update README.md to reflect the new command

Jerome-Joseph-1 commented on src/scrapper.py:
```diff
@@ -158,6 +158,24 @@ def get_next_number(number):
+    else:
+        logger.error(f"Failed: Retrying with new Captcha ... Attempt {failureCount + 1}")
+        cookie, token, captchaCode = init_connection(number)
+        failureCount += 1
```
Is the 'failureCount' variable used to retry data fetching if it fails? I can't find where it's reused for fetching.

Colin-Christon replied:
I have updated the code such that based on the `failureCount` variable atmost 5 times we can retry the failed result.

Jerome-Joseph-1 replied:
Alright, Thanks!

Colin-Christon committed:
Address feedback
Add retry condition if failure in fetching result
Failure count is atmost 5
Updated if condition to check failureCount in fetch_result method

Colin-Christon committed:
Update while loop condition
